validate.go

```diff
@@ -170,6 +170,17 @@ func checkProcess(process rspec.Process, rootfs string) {
 		}
 	}
 
+	if len(process.Args) < 1 {
+		logrus.Fatalf("args should not be empty")
+	}
+	command := process.Args[0]
+	if filepath.IsAbs(command) {
+		cmdPath := path.Join(rootfs, command)
+		if _, err := os.Stat(cmdPath); err != nil {
+			logrus.Fatalf("Cannot find the command path %q", cmdPath)
```

Comment:
@wking  Thanks for you information. I think you are right. I will remove code which checks whether command exists or not
From runtime-sepc, it seems args is required not optional. So I think checking whether args is empty is necessary.